DEV_README.md

```diff
@@ -0,0 +1,26 @@
+# Skadi Chart
```

Comment:
Can you add a link to this DEV_README from the main README, to make it more discoverable?